.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,39 @@
+name: e2e-mock-provider-tests
+on: 
+  workflow_dispatch:
+    inputs:
+      registry:
+        description: 'Registry'
+        required: true
+        default: 'gcr.io/k8s-staging-csi-secrets-store'
+      driverImage:
+        description: 'Secret Store CSI driver image'
+        required: true
+        default: 'driver'
+      driverImageVersion:
```

Comment:
We build and publish crd image to staging registry too and will want to test it. Should we add them to the input options here too?

Reply:
added